### PR TITLE
CR-1026218 Need PIDs of processes running on device

### PR DESCRIPTION
**Pull request by maxzhen**
@sonals , please take a look and see if this is what you have in mind.

### CODE REVIEW

File: src/runtime_src/driver/xclng/drm/xocl/userpf/xocl_sysfs.c [hunk 1]
```diff
@@ -51,21 +51,28 @@ static ssize_t user_pf_show(struct device *dev,
...
-			   xclbin_id ? xclbin_id : 0,
-			   atomic_read(&xdev->outstanding_execs),
-			   atomic64_read(&xdev->total_execs),
-			   get_live_client_size(xdev));
+	size += sprintf(buf + size, "xclbin:\t\t\t%pUb\n",
+		xclbin_id ? xclbin_id : 0);
+	size += sprintf(buf + size, "outstanding execs:\t%d\n",
+		atomic_read(&xdev->outstanding_execs));
+	size += sprintf(buf + size, "total execs:\t\t%ld\n",
+		atomic64_read(&xdev->total_execs));
```
Comment (sonals) [inline src/runtime_src/driver/xclng/drm/xocl/userpf/xocl_sysfs.c]: I wonder if should keep the original contexts and then add the pids in the following line?

**Comment by maxzhen** [comment]
@sonals , pushed a change to fix this. please take a look.

**Comment by keryell** [comment]
Just curious about the feature implemented here. Is it something similar to running `lsof` on the device driver controlling the card?

**Comment by maxzhen** [comment]
> Just curious about the feature implemented here. Is it something similar to running `lsof` on the device driver controlling the card?

Yes.

**Pull request merged**